mitodl/ocw-hugo-projects
# old course theme removed

#### Pre-Flight checklist


#### What are the relevant tickets?
Related to https://github.com/mitodl/ocw-hugo-themes/issues/986

#### What's this PR do?
- Removes old course theme

#### How should this be manually tested?
- Point your `COURSE_HUGO_CONFIG_PATH` env var in `ocw-hugo-themes` to `course/config.yaml` instead of `course-v2/config.yaml`
- Build any course locally
- Verify that the course builds successfully
